package.json

```
@@ -4,7 +4,7 @@
   "description": "A powerful library for interacting with the Discord API",
   "scripts": {
     "test": "npm run lint && npm run docs:test && npm run lint:typings && npm run test:typescript",
-    "test:typescript": "tsc --noEmit",
+    "test:typescript": "npx tsd",
```

```suggestion
    "test:typescript": "tsc --noEmit && tsd",
```

At this point we might as well just do this and don't change the workflow at all.